index.html

```diff
@@ -2469,23 +2476,26 @@ <h4>parseDerivedProofValue</h4>
           <ol class="algorithm">
             <li>
 Ensure the |proofValue| string starts with `u`, indicating that it is a
-multibase-base64url-no-pad-encoded value, throwing an error if it does not.
+multibase-base64url-no-pad-encoded value, if it does not an error MUST be raised
+and SHOULD convey an error type of `PROOF_VERIFICATION_ERROR`.
             </li>
             <li>
 Initialize |decodedProofValue| to the result of base64url-no-pad-decoding the
 substring after the leading `u` in |proofValue|.
             </li>
             <li>
 Ensure that the |decodedProofValue| starts with the ECDSA-SD disclosure proof
-header bytes `0xd9`, `0x5d`, and `0x01`, throwing an error if it does not.
+header bytes `0xd9`, `0x5d`, and `0x01`, if it does not an error MUST be raised
+and SHOULD convey an error type of `PROOF_VERIFICATION_ERROR`.
             </li>
             <li>
 Initialize |components| to an array that is the result of CBOR-decoding the
 bytes that follow the three-byte ECDSA-SD disclosure proof header. Ensure the
 result is an array of five elements. Ensure the result is an array of five
 elements: a byte array of length 64, a byte array of length 36, an array of byte
 arrays, each of length 64, a map of integers to byte arrays of length 32, and an
```

Comment:
In the `map of integers to byte arrays`, is it `the map` that is `of length 32`, or is it the `byte arrays` that are `of length 32`? I'll revise this suggestion, if I've guessed wrong.
```suggestion
Initialize |components| to an array that is the result of CBOR-decoding the bytes
that follow the three-byte ECDSA-SD disclosure proof header. If the result is not
an array of the following five elements — a byte array of length 64; a byte array
of length 36; an array of byte arrays, each of length 64; a map of integers to
byte arrays, each of length 32; and an array of integers — an error MUST be raised
and SHOULD convey an error type of `PROOF_VERIFICATION_ERROR`.
```